### PR TITLE
create section for directory structure explanation

### PR DESCRIPTION
**Pull request by lucaslago**

### CODE REVIEW

File: monitoring/README.md [hunk 1]
```diff
@@ -28,6 +28,35 @@ The components are going to be available at:
...
+  ├── log
+  │   ├── empty
+  └── logstash
+  │   ├── logstash.conf
+  │   └── template.json
+```
+
+* docker-compose.yml - Docker Compose configuration file
+* /filebeat
+  *  filebeat.yml - Filebeat configuration file, passed as volume to
```
Comment (waldemarnt) [inline monitoring/README.md]: can we link filebeat official documentation here?

File: monitoring/README.md [hunk 2]
```diff
@@ -28,6 +28,35 @@ The components are going to be available at:
...
+     Filebeat container
+* /jmeter
+  * monitoring-load.jmx - Apache Jmeter exported configuration
+* /log - Folder that Filebeat watches, .log files should be generated
+  here
+  * empty - Empty file just to make sure git tracks this directory
+* /logstash - This entire folder is passed as volume to Logstash
```
Comment (waldemarnt) [inline monitoring/README.md]: can we link logstash official documentation here?

**Comment by jeffhsta** [comment]
:+1:

File: monitoring/README.md (hunk 1 above)
Comment (lucaslago) [inline monitoring/README.md]: we could link logstash's as well

**Comment by glauco** [comment]
LGTM!

**Pull request merged**